fasta2a/fasta2a/_otel.py

```diff
@@ -0,0 +1,120 @@
+from __future__ import annotations as _annotations
+
+import contextvars
+from collections.abc import AsyncIterable
+from dataclasses import dataclass
+from typing import Any, Generic, TypeVar
+
+import anyio
+from anyio.streams.memory import MemoryObjectReceiveStream, MemoryObjectSendStream
+
+try:
+    from opentelemetry import context as otel_context
+    from opentelemetry.trace import INVALID_SPAN, Span, get_current_span, set_span_in_context
+
+    T = TypeVar('T')
+
+    @dataclass
+    class WithContext(Generic[T]):
```

Comment:
Are these classes used in other files? If so, should they be defined in the `except:` branch? If not, should they be made private?

Reply:
that file is not used. I'm deleting it.